src/rdkafka_broker.c

```diff
@@ -3830,14 +3852,32 @@ static int rd_kafka_broker_fetch_toppars (rd_kafka_broker_t *rkb, rd_ts_t now) {
 
         if (unlikely(rkb->rkb_active_toppar_cnt == 0))
                 return 0;
-
+#if WITH_ZSTD
 	rkbuf = rd_kafka_buf_new_request(
```

Comment:
We'll want to use the latest supported (by both sides) fetch version. So do something like this:
```
        ApiVersion = rd_kafka_broker_ApiVersion_supported(
                rkb, RD_KAFKAP_Fetch, 0, 10, NULL);
        if (ApiVersion < 10 && ApiVersion > 4)
            ApiVersion = 4;  /* We support 0..4, 10, skip intermediates. */
```